contracts/Recruitment.sol

```diff
@@ -18,6 +18,26 @@ contract Recruitment {
     owner = msg.sender;
   }
 
+  struct ReferralScore {
+        uint256 score;  //Score given by the hiring company to the candidate 
+        address senderAddress; // Wallet address of the hiring company
+  }
+  event ReferralScoreSubmitted(address senderAddress, address referrerWallet, string referrerEmail, uint256 score);
+  mapping(address => ReferralScore[]) public referralScores;
+
+  function submitReferralScore(uint256 score,address senderAddress,address referrerWallet,string memory referrerEmail) public returns (bytes32) {
```

Comment:
Hi we should remove this parameters: 
referrerEmail: remove completely as we don't need it. You can remove this from the related objects
senderAddress: remove it from input parameters as it can be replaced by msg.sender
...from method: submitReferralScore

Does this make sense?